src/Kernels/Receiver.cpp

```diff
@@ -116,97 +156,111 @@ double seissol::kernels::ReceiverCluster::calcReceivers(  double time,
   }
 #endif
 
-  double receiverTime = time;
   if (time >= expansionPoint && time < expansionPoint + timeStepWidth) {
-    for (size_t i = 0; i < m_receivers.size(); ++i) {
+    // heuristic; to avoid the overhead from the parallel region
+    auto threshold = omp_get_num_threads() * 100;
```

Comment:
what about: std::max(1000, omp_get_num_threads() * 100) 
This would ensure that even if the number of threads is low, parallelization doesn't kick in